bin/install

```diff
@@ -488,7 +488,7 @@ initiate() {
 
     echo
 
-    if [ "$yes_g" != "y" ]; then
+    if [ "${yes_g,,}" != "y" ]; then
```

Comment:
This is not POSIX syntax ([see CI](https://github.com/stefanmaric/g/actions/runs/11217159953/job/31179515720?pr=40)) (and thank God, because this is super hard to ready.

Would suggest a simple OR operator.